src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1078,7 +1078,7 @@ This function accepts the following properties:
 
 <p><Since v="2.5.0" /></p>
 
-**Type:** `'content' | 'data'`  
+**Type:** `'content' | 'data'`
```

Comment:
![image](https://github.com/withastro/docs/assets/25167721/a991229d-affe-48b1-8ef1-bbdd59a6b948)

Hey  @Princesseuh These changes break the format. All the line breaks are broken. The original format seems intentional.

Reply:
Arf, I hate Markdown, auto formatting always breaks everything. I'll fix it

Reply:
Fixed!